client/src/components/trace-table/tableList.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+
+type TraceData = {
+  timestamp: string,
+  traceId: string,
+  responseTime: string,
+  response: string,
+  method: string,
+  url: string,
+  namespaces: string,
+}
+
+/** 
+   * Renders the list of elements contained within the Trace Table Drawer
+   * @Remarks Pre-fetching when hovering over drawerTabHandle will greatly improve performance
+   */
+const tableList = () => {
+  
```

Comment:
Maybe something to implement for the next sprint, consider making a ticket on notion!